app/alert_screen.py

```diff
@@ -0,0 +1,247 @@
+# Copyright (C) 2021, Pyronear contributors.
+
+# This program is licensed under the Apache License version 2.
+# See LICENSE or go to <https://www.apache.org/licenses/LICENSE-2.0.txt> for full license details.
+
+"""
+The following Python file is dedicated to the alert screen of the web application.
+The alert screen corresponds to a web page which will be displayed on a big screen in the CODIS room. There will be no
+interaction with the user. The main use of this page is to display a "sober" screen when there are no alerts. When an
+alert pops out, the screen will automatically change to display various information.
+
+Most functions defined below are designed to be called in the main.py file.
+"""
+
+# ----------------------------------------------------------------------------------------------------------------------
+# IMPORTS
+
+# Various modules provided by Dash to build the page layout
+import dash_bootstrap_components as dbc
+import dash_core_components as dcc
+import dash_html_components as html
+
+
+# Importing the pyro-API client
+
+
+# ----------------------------------------------------------------------------------------------------------------------
+# CONTENT
+def no_alert_detected_screen():
```

Comment:
I am going to be a pain but do you think we could rename the function something like "build_no_alert_detected_screen" (or equivalent) to ensure consistency in the code?

Reply:
done